src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml

```diff
@@ -10,10 +10,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <title>@HeadTitle</title>
-    <meta name="description" content=@HeadDescription />
-    <meta property="og:image" content="https://www.devadventcalendar.pl/theme-2019/images/dac2019-social.png">	    <meta property="og:image" content="https://www.devadventcalendar.pl/theme-2019/images/dac2019-social.png" />
-    <meta property="og:title" content=@HeadTitle />
-    <meta property="og:description" content=@HeadDescription />
+    <meta name="description" content="DevAdventCalendar - inicjatywa łącząca tradycję kalendarza adwentowego z programowaniem. Celem jest stworzenie konkursu dla osób związanych ze światem IT. " />
+    <meta property="og:image" content="https://www.devadventcalendar.pl/theme-2019/images/dac2019-social.png">
+    <meta property="og:image" content="https://www.devadventcalendar.pl/theme-2019/images/dac2019-social.png" />
+    <meta property="og:title" content="DevAdventCalendar - programistyczny kalendarz adwentowy" />

```

Comment:
I didnt translate text within <meta> tags because its for search engines and when I tried to use Resources here there was problem with polish encodings

Reply:
ok :)